api-report/merge-tree.api.md

```diff
@@ -343,6 +340,22 @@ export interface IAttributionCollectionSpec<T> {
     }>;
 }
 
+// @public
+export interface IAttributionPolicyFactory {
```

Comment:
still need to do a bit of cleanup of API tags and docs in this realm.

Also not totally convinced I want to expose classes for the policy registry elements or have them be self-named, but that's a bit more minor (self-naming is less of a concern for closed-ecosystem set of options, which is what attribution policies currently are. Maybe we open them up in the future but it's pretty easy to shoot yourself in the foot, so I'd rather not if we can get away with it)